channel-manager-api.md

```diff
@@ -34,27 +34,50 @@ Mews automatically sends changes in Inventory \(once connection is set up\). Dat
 
 #### Request `[ChannelManagerApiAddress]/updatePrices`
 
+*Mews always pushes both `gross` and `net` prices. Correct value needs to be picked up by a channel manager. 
+*`amount`to be deprecated in May 2020.
+
 ```javascript
 {
    "clientToken": "[Mews Client token]",
    "connectionToken": "[Token of a concrete connection]",
-   "ratePrices": [
-      {
-         "spaceTypeCode": "QD",
-         "ratePlanCode": "FF",
-         "from": "2020-01-01",
-         "to": "2020-01-31",
-         "prices": [
-            {
-               "guestCount": 1,
-               "amount": 95,
-               "currencyCode": "USD"
-            },
-            {
-               "guestCount": 2,
-               "amount": 100,
-               "currencyCode": "USD"
-            }
+   {
+    "spaceTypeCode": "D1",
+    "ratePlanCode": "FF",
+    "prices": [
+        {
+         ~~"amount": 100.00,~~
```

Comment:
As discusssed ~~ should be removed and used only in table part

Reply:
done